app/segments/drill/[id]/submission/input.js

```diff
@@ -631,18 +620,24 @@ export default function Input({ setToggleResult, setOutputData }) {
       );
 
       setOutputData(outputData);
-      uploadAttempt(
-        outputData,
-        currentUserId,
-        assignedTime,
-        drillId,
-        drillInfo,
-        currentLeaderboard,
-      ).then(() => {
+      try {
```

Comment:
Will this try catch block be able to get errors from all the functions underneath?

Reply:
yes. You can try it out by putting a throw in one of the inner function